src/datasource/components/Logs/LogsCell.tsx

```diff
@@ -0,0 +1,325 @@
+import React from 'react'
+import { Log } from 'datasource/types'
+import { LogColumnType, LogColumn } from 'datasource/components/Logs/types'
+import  getLogTableContext from 'datasource/components/Logs/context'
+import { Button, useTheme2 } from '@grafana/ui'
+import { getTimeZone } from '@grafana/data'
+import { DateTime } from 'luxon'
+
+
+interface LogKeyValProps {
+    field: string,
+    val: any
+}
+
+
+const LogKeyVal = ({ field, val }: LogKeyValProps) => {
+    return (<div style={{display:'inline-block', paddingRight: '10px'}}>
+        <div style={{backgroundColor: useTheme2().isDark ? '#343741' : '#e6f1fa', display: 'inline', borderRadius: '6px', marginRight: '3px', padding: '2px 4px'}}>
```

Comment:
Todo - we should move the theme colors to a separate file for common use across the components.